MCprep_addon/materials/generate.py

```diff
@@ -30,10 +30,18 @@
 
 AnimatedTex = Dict[str, int]
 
-class PackFormat(Enum):
-	SIMPLE = 0
-	SEUS = 1
-	SPECULAR = 2
+class DocEnum(Enum):
+	def __new__(cls, value, doc=""):
+		self = object.__new__(cls)  # calling super().__new__(value) here would fail
+		self._value_ = value
+		if doc != "":
+			self.__doc__ = doc
+		return self
+    
+class PackFormat(DocEnum):
+	SIMPLE = 0, "Use a simple shader setup with no PBR or emission falloff."
+	SEUS = 1, "Sets the pack format to Specular."
+	SPECULAR = 2, "Sets the pack format to SEUS."
```

Comment:
Why? 

You're taking a simple-to-understand concept (enums) and overcomplicating them with something related to docstrings. I can't even understand what's going on here, how would a new developer figure out what on God's earth this means?